intel-sgx/pcs/src/qe_identity.rs

```diff
@@ -226,6 +231,15 @@ where
     serializer.serialize_str(&miscselect)
 }
 
+fn datetime_serializer<S>(timestamp: &DateTime<Utc>, serializer: S) -> ::std::result::Result<S::Ok, S::Error>
```

Comment:
Can you put this close to `iso8601timestamp` and use the same format string constant in both? `#[serde(with = "module")]` may be a good way to do this.